app/components/inputgroup/inputgroup.js

```diff
@@ -191,6 +196,42 @@ var InputGroup = React.createClass({
     );
   },
 
+  renderSelect: function() {
+    var self = this;
+    var placeholder = this.props.value === this.props.placeholder;
```

Comment:
actually, since none of the places are using the `self` in the way that it's classically used (in order to get around internal function scope) it's probably better to not define a `self` here at all - will fix.